util.c

```diff
@@ -2377,23 +2377,23 @@ Perl_my_popen_list(pTHX_ const char *mode, int n, SV **args)
     /* If we managed to get status pipe check for exec fail */
     if (did_pipes && pid > 0) {
 	int errkid;
-	unsigned n = 0;
+	unsigned this_n = 0;
```

Comment:
This is the total number of bytes read from the pipe to the child, so maybe `read_total`.

Reply:
> This is the total number of bytes read from the pipe to the child, so maybe `read_total`.

In the smoke-me/jkeenan/local-variable-warnings branch, I have implemented your suggestions in this and the preceding three comments.  See commits 2ed84d2c0250c70276c76ee7fda52acf2cf5bed4 and 3d0e1729f00630893c1af2fb5ba59fda61fe45ba.

Thank you very much.
Jim Keenan